authors/apps/authentication/models.py

```diff
@@ -1,18 +1,20 @@
 import jwt
+import os
 
 from datetime import datetime, timedelta
 
 from django.conf import settings
-from django.contrib.auth.models import (
-    AbstractBaseUser, BaseUserManager, PermissionsMixin
-)
+from django.contrib.auth.models import (AbstractBaseUser, BaseUserManager,
+                                        PermissionsMixin)
 from django.db import models
+from authors.apps.core.token import encode_auth_token
```

Comment:
Is `encode_auth_token` being used in this file??

Reply:
No, it's not used, I was trying to add token field but I wasn't able